src/domains/domu/domu_runner.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef DOMAIN_U_RUNNER_H_
+#define DOMAIN_U_RUNNER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int domu_start();
```

Comment:
Logically it has to be called somewhere under CONFIG_AOS_ENABLE_DOMU ifdef.

Reply:
Sry. missed it

Reply:
`int domu_start(void);` probably. Looking at `#ifdef __cplusplus` you expect that this code *might* be compiled with plain C, so it should conform plain C declaration.